### DISCUSSION TITLE
IPv6 Check?

### DISCUSSION THREAD

**Discussion by jakevis** [upptime/upptime#308] [Q&A]
Hi there,

I am trying to find a way to check some of my IPv6 services. Tried a couple things - tcp ping using [ipv6::] notation as well as jus the IPv6 address. Also tried a https check to url: https://ipv6.google.com

No luck with any of them... is IPv6 supported? any workarounds?

Cheers,

Jake

https://github.com/emeraldonion/status/

**Comment by xs314** [comment]
I support it!Although the machine of the Github Actions not supported ipv6,we can use cloudflare warp.😋
